docs/usage/dns-autoscaling.md

```diff
@@ -0,0 +1,63 @@
+# DNS Autoscaling
```

Comment:
Please add this document to the index (README.md)

Reply:
Done with newest commit.